pom.xml

```diff
@@ -197,6 +197,12 @@ SOFTWARE.
         <version>3.0.1u2</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>org.eolang</groupId>
+        <artifactId>jucs</artifactId>
+        <version>0.0.2</version>
```

Comment:
@yegor256 Maybe the first version have to be at least `0.1.2`? I'm not sure here but in the semver the third number is just a [bugfix](https://semver.org). I read that version as "bug fix for 0 minor version" - sounds strange.